packages/lesswrong/components/sunshineDashboard/SunshineReportedItem.jsx

```diff
@@ -27,20 +27,34 @@ class SunshineReportedCommentsItem extends Component {
     const {
       currentUser,
       report,
-      editMutation,
+      updateComment,
+      updatePost,
       reportEditMutation,
     } = this.props
-    if (confirm("Are you sure you want to immediately delete this comment?")) {
-      editMutation({
-        documentId: report.comment && report.comment._id,
-        set: {
-          deleted: true,
-          deletedDate: new Date(),
-          deletedByUserId: currentUser._id,
-          deletedReason: "spam"
-        },
-        unset: {}
-      })
+    if (confirm(`Are you sure you want to immediately delete this ${report.comment ? "comment" : "post"}?`)) {
+      if (report.comment) {
+        updateComment({
+          documentId: report.comment._id,
+          set: {
+            deleted: true,
+            deletedDate: new Date(),
+            deletedByUserId: currentUser._id,
+            deletedReason: "spam"
```

Comment:
I don't think this should change right now, but I notice that:

a) our workflow should be to basically always mass-purge users who are spamming
b) therefore, the "delete individual post/comment" option should probably be used for things that aren't spam, but instead bad for other reasons, so auto-labeling it "Spam" is a bit weird.